src/browser/components/preferences/main-js.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/browser/components/preferences/main.js b/browser/components/preferences/main.js
-index 46283d3eacfea0f419afa3bdfd2a51e411502a20..4fe229706eb0145a21042b8b3cca323682e89e76 100644
+index 46283d3eacfea0f419afa3bdfd2a51e411502a20..730058671fb0cbab0b5b5b1c01cdb912560116a2 100644
 --- a/browser/components/preferences/main.js
 +++ b/browser/components/preferences/main.js
-@@ -1117,6 +1117,11 @@ let SETTINGS_CONFIG = {
+@@ -102,6 +102,7 @@ Preferences.addAll([
+   { id: "browser.warnOnQuitShortcut", type: "bool" },
+   { id: "browser.tabs.warnOnOpen", type: "bool" },
+   { id: "browser.ctrlTab.sortByRecentlyUsed", type: "bool" },
++  { id: "browser.ctrlTab.cycleByAttribute", type: "bool" },
```

Comment:
This can be done in zen-settings.js:

https://github.com/zen-browser/desktop/blob/5bd0a01d5369d3bc5cc3f2481319b997c0ca703e/src/browser/components/preferences/zen-settings.js#L1057-L1138